src/main/java/org/snomed/snowstorm/core/data/services/BranchMetadataKeys.java

```diff
@@ -11,4 +11,5 @@ public interface BranchMetadataKeys {
 	String SHORTNAME = "shortname";
 	String DEPENDENCY_RELEASE = "dependencyRelease";
 	String PREVIOUS_RELEASE = "previousRelease";
+	String UPDATE_MRCM_DOMAIN_TEMPLATES = "updateMRCMDomainTemplates"; // Toggle whether MRCM Domain Templates are updated during commit.
```

Comment:
If this flag is set in the branch metadata, both the attribute rules and domain templates will be updated like normal. I've added this in case we have a project where the user doesn't want to have to manually trigger the update of the domain templates file.